blog/2023-07-27-introducing-substratus.md

```diff
@@ -0,0 +1,137 @@
+---
+slug: introducing-substratus
+title: Introducing Substratus
+authors:
+  name: Brandon Bjelland
+  title: Co-founding Engineer, CXO
+  url: https://github.com/brandonjbjelland
+  image_url: https://avatars.githubusercontent.com/u/2502520?v=4
+tags: [hello world, introduction, oss launch]
+---
+# Announcing Substratus: A Multi-cloud Substrate for AI
+
+We're beyond excited to announce Substratus (s8s), an open-source platform to
+manage machine learning primitives in Kubernetes.
+
+Open source machine learning models are exploding in popularity and it's nearly
+impossible for any individual or organization to keep up. We felt this exact
+pain in early 2023:
+
+* How can we easily experiment with LLMs? 👩‍🔬
+* Can I inference using my workstation's hardware and OS? 💻
+* What about fine-tuning, can I do that locally? Why do I run out of GPU
+  memory?! 📏
+* Can I stream or do I really need to download this whole dataset?! I didn't
+  plan on uninstalling Flight Simulator tonight...✈️
+* Is any part of this workflow portable or repeatable? ♻️
+
+It would be reasonable to walk away from a night of yak-shaving with a Colab
+subscription and a handful of modest wins. But your plan to deliver a secure,
+consistent, and portable ML workflow to your day job is still effectively
+🤷🏾‍♀️. Even through great effort and ample research, your ultimate strategy
+might be: **hope**, that your engineering team is satisfied with Sagemaker's
+technical choices and your CFO doesn't care about pricing. 💸
```

Comment:
I think this paragraph is slightly too long. I do like the points made, I'm just afraid the reader might never get to the Why Substratus part.

Reply:
Totally fair. The scenario runs a bit long. I'll aim to chop by at least 1/3